src/blog/blog.controller.ts

```diff
@@ -40,4 +40,9 @@ export class BlogController {
       }
     }
   }
+
+  @Delete('/delete/:id')
```

Comment:
P1: 이미 delete method로 삭제라는 기능이 표현되어 있는데 또 delete가 들어간 것은 redundant 합니다. 
```suggestion
  @Delete('/:id')
```
가 더 나은 것 같습니다.

Reply:
수정 완료했습니다!